README.md

```diff
@@ -98,6 +98,12 @@ CLI options:
 --scope                     Restricts the command to packages which names match the given glob pattern.
 
                             Default: null
+                            
+--packages-prefix           The common name of the packages. The prefix will be removed from packages' names in order to
```

Comment:
Why is this a CLI config option? Do you expect anyone to use it? I don't. If anywhere – this is for `mgit.json` only.